docs/Running_Rinkeby_Nitro_Node.md

```diff
@@ -0,0 +1,90 @@
+---
+id: Running_Rinkeby_Nitro_Node
+title: Running full Nitro node for Arbitrum Rinkeby TestNet after Nitro upgrade
+sidebar_label: Running a Rinkeby Nitro Node
+---
+
+Note: Rinkeby Nitro will not work until after the Nitro upgrade scheduled for Thursday, July 28th.
```

Comment:
Do you mean Arbitrum Classic nodes on Rinkeby will stop working?

Reply:
updated to be clearer